### PR TITLE
Change label in Request housing form

### LINKED ISSUE #147: [Must have] Change label in Request housing form
please change this label "Te rugăm să ne oferi mai multe detalii referitoare la cazul pe care îl supui atenției noastre!" to "Te rugăm să ne oferi mai multe detalii despre situatia ta ca sa te ajutam cat mai repede"

### LINKED ISSUE #148: [Must Have] Add info in the Terms and conditions page in admin dashboard
The text displayed in the first step of every form (the legal conditions to offer or request accommodation) need to also be displayed in the Admin dashboard so that when the user logs in has acces to the information provided here. The info should be available in all four languages. 

Here is the [info that needs to be copied
](http://un-acoperis.heroesof.tech/ro/request-help)
Here is where it has to be displayed
image

### PR DESCRIPTION
**Pull request by danut3101**
### Requirements for making a pull request

Thank you for contributing to our project! 

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.

### What does it fix?

Closes #147 

Please mention the main changes this PR brings.

### How has it been tested?

Please describe the tests that you ran to verify your changes.

### CODE REVIEW

**Comment by danut3101** [comment]
Este, de fapt pentru #147

File: resources/views/host/home.blade.php [hunk 1]
```diff
@@ -3,8 +3,6 @@
 @section('content')
     @include('partials.info-box', [
         'title' => __('Terms and conditions'),
-        'text' => __('host.general.terms.text'),
-        'buttonText' => __('View Terms And Conditions'),
-        'buttonUrl' => route('terms-host')
+        'text' => $termsAndConditionsForHosts
```
Comment (gheorghelupu17) [inline resources/views/host/home.blade.php:6]: Lasă butonul sa ducă aici te rog http://un-acoperis.heroesof.tech/host/accommodation/add

File: resources/views/refugee/home.blade.php [hunk 2]
```diff
@@ -3,8 +3,6 @@
 @section('content')
     @include('partials.info-box', [
         'title' => __('Terms and conditions'),
-        'text' => __('refugee.general.terms.text'),
-        'buttonText' => __('View Terms And Conditions'),
-        'buttonUrl' => route('terms-refugee')
+        'text' => $termsAndConditionsForRefugees
```
Comment (gheorghelupu17) [inline resources/views/refugee/home.blade.php:6]: Idem ca mai sus lasă butonul sa ducă la formular te rog

**Review by gheorghelupu17 — DISMISSED** [review]

File: resources/views/refugee/home.blade.php (hunk 2 above)
Comment (danut3101) [reply resources/views/refugee/home.blade.php:6]: LA fel ca mai sus...

File: app/Http/Controllers/Host/ProfileController.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers\Host;
 
+use A17\Twill\Repositories\SettingRepository;
```
Comment (andreiio) [inline app/Http/Controllers/Host/ProfileController.php:5]:
```suggestion
```

File: app/Http/Controllers/Host/ProfileController.php [hunk 4]
```diff
@@ -22,9 +23,10 @@ class ProfileController extends Controller
     /**
      * @return View
      */
-    public function home(): View
+    public function home(SettingRepository $settingRepository): View
```
Comment (andreiio) [inline app/Http/Controllers/Host/ProfileController.php:26]:
```suggestion
    public function home(): View
```

File: app/Http/Controllers/Host/ProfileController.php [hunk 5]
```diff
@@ -22,9 +23,10 @@ class ProfileController extends Controller
...
     {
-        return view('host.home');
+        return view('host.home')
+            ->with('termsAndConditionsForHosts', $settingRepository->byKey('terms_and_conditions_for_hosts') ?? '');
```
Comment (andreiio) [inline app/Http/Controllers/Host/ProfileController.php:29]:
```suggestion
        return view('host.home');
```

File: app/Http/Controllers/Refugee/ProfileController.php [hunk 6]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers\Refugee;
 
+use A17\Twill\Repositories\SettingRepository;
```
Comment (andreiio) [inline app/Http/Controllers/Refugee/ProfileController.php:5]:
```suggestion
```

File: app/Http/Controllers/Refugee/ProfileController.php [hunk 7]
```diff
@@ -15,10 +16,10 @@ class ProfileController extends Controller
 {
     const PER_PAGE = 5;
 
-    public function home(): View
+    public function home(SettingRepository $settingRepository): View
```
Comment (andreiio) [inline app/Http/Controllers/Refugee/ProfileController.php:19]:
```suggestion
    public function home(): View
```

File: app/Http/Controllers/Refugee/ProfileController.php [hunk 8]
```diff
@@ -15,10 +16,10 @@ class ProfileController extends Controller
...
     {
-
-        return view('refugee.home');
+        return view('refugee.home')
+            ->with('termsAndConditionsForRefugees', $settingRepository->byKey('terms_and_conditions_for_refugees') ?? '');
```
Comment (andreiio) [inline app/Http/Controllers/Refugee/ProfileController.php:22]:
```suggestion
        return view('refugee.home');
```

File: resources/views/host/home.blade.php (hunk 1 above)
Comment (andreiio) [inline resources/views/host/home.blade.php:6]:
```suggestion
        'text' => __('host.general.terms.text'),
        'buttonText' => __('View Terms And Conditions'),
        'buttonUrl' => route('terms-host')
```

File: resources/views/refugee/home.blade.php (hunk 2 above)
Comment (andreiio) [inline resources/views/refugee/home.blade.php:6]:
```suggestion
        'text' => __('refugee.general.terms.text'),
        'buttonText' => __('View Terms And Conditions'),
        'buttonUrl' => route('terms-refugee')
```

**Review by andreiio — CHANGES_REQUESTED** [review]
Changes unrelated to the issue shouldn't be part of this PR

File: resources/lang/ro.json [hunk 9]
```diff
@@ -435,7 +435,7 @@
     "I have dependants expl": "Bifează căsuța dacă ești împreună cu familia, minori/bătrâni care necesită asistența ta, pentru a completa datele acestora și a prioritiza găsirea unei locuințe comune.",
     "current_location_placeholder": "Botoșani, județ Botoșani, România",
     "special_request_placeholder": "Scaun cu rotile, spațiu de cazare la parter",
-    "Please give us more details about the case you are bringing to our attention": "Te rugăm să ne oferi mai multe detalii referitoare la cazul pe care îl supui atenției noastre!",
+    "Please give us more details about the case you are bringing to our attention": "Te rugăm să ne oferi mai multe detalii despre situația ta ca sa te ajutăm cât mai repede.",
```
Comment (andreiio) [inline resources/lang/ro.json:438]:
```suggestion
    "Please give us more details about the case you are bringing to our attention": "Te rugăm să ne oferi mai multe detalii despre situația ta ca să te ajutăm cât mai repede.",
```

**Review by andreiio — APPROVED** [review]

File: app/Http/Controllers/Refugee/ProfileController.php [hunk 10]
```diff
@@ -17,7 +17,6 @@ class ProfileController extends Controller
 
     public function home(): View
     {
-
         return view('refugee.home');
```
Comment (andreiio) [inline app/Http/Controllers/Refugee/ProfileController.php:20]:
```suggestion

        return view('refugee.home');
```

**Pull request merged**